contracts/core/Allo.sol

```diff
@@ -1,128 +1,70 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity 0.8.19;
 
+import "./IAllo.sol";
+
 import "@openzeppelin-upgradeable/proxy/utils/Initializable.sol";
 import "@openzeppelin-upgradeable/token/ERC20/IERC20Upgradeable.sol";
-import "@openzeppelin-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
-import {AccessControl} from "@openzeppelin/access/AccessControl.sol";
+import "@openzeppelin/access/AccessControl.sol";
```

Comment:
Why use named imports on some and not all? Is there an advantage to this?

Reply:
named imports doesn't make sense/there is no big difference in our case.
maybe it's ncer to read for some ppl? idk :D

Reply:
Just for reading and consistency IMO either way is fine.